components/checkbox.js

```diff
@@ -38,7 +38,7 @@ const StyledInput = styled("input")(
 );
 
 const StyledLabel = styled("span")({
-  fontFamily: globalTheme.fontFamilySerif,
+  fontFamily: globalTheme.fontFamilySansSerif,
```

Comment:
I realized the fonts here in both checkbox and radio weren't following the designs, so that's why I applied this change to all instances of the checkboxes and radio buttons

Reply:
👍